xbmc/network/WebServer.cpp

```diff
@@ -868,6 +871,11 @@ int CWebServer::CreateErrorResponse(struct MHD_Connection *connection, int respo
         payloadSize = strlen(NOT_SUPPORTED);
         payload = (const void *)NOT_SUPPORTED;
         break;
+
+      case MHD_HTTP_SERVICE_UNAVAILABLE:
+        payloadSize = strlen(SERVICE_UNAVAILABLE);
+        payload = (const void *)SERVICE_UNAVAILABLE;
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types


> New code has to use C++ style casts and not older C style casts. [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#122-casts